### DISCUSSION TITLE
Too many branches

### DISCUSSION THREAD

**Discussion by tiagonmas** [banco-alimentar/alimentestaideia.pt#201] [Q&A]
Maybe we should do a "branch cleanup party" :) ?! @guerrerotook @joalmeid

**Comment by guerrerotook** [comment]
That is right. In one point we should merge my original branch with the developer. Replace the original code with the dotnet version as well.

**Comment by joalmeid** [comment]
Yep, totally agree. Actually mention not being able to delete branches to Luis.
As it is now protected, we cant delete them - and we shouldnt - but lets clean it!.
